### DISCUSSION TITLE
Changing the function in the COCO example for the approximation's algorithm.

### DISCUSSION THREAD

**Discussion by EdernOllivier** [numbbo/coco#2138] [Ideas]
Dear people,

I do believe in the COCO platform for finding the approximation of a function in a few step.
However, I do need to change the given function to get the values of each step with the uniform convergence.
Can you integer the functions of the glibc bib. to do it ?

Best regards,

Edern Ollivier.

**Comment by brockho** [comment]
Dear Edern,

Do we understand correctly that you want to integrate new functions into the COCO platform? If not, please give a few more details, maybe with an example such that we can help.

**Reply by EdernOllivier** [reply]
Dear Dimo,

No, I only want to make a known entire serie to converge with COCO.
I have tried with the sum of 1 /n^2 to get the value of Pi^2 /6, but it hasn't given the good value.
Now I should print the values of x at the screen to see the bug.

Best regards,

Edern Ollivier.

**Reply by brockho** [reply]
Hi. Would you be so kind and send us your current code? It will be much easier to help you if we understand what you want to achieve with COCO.

**Reply by EdernOllivier** [reply]
Hi Dimo. Right, I have changed the values. I doesn't succeed.
[example_experiment.zip](https://github.com/numbbo/coco/files/9658169/example_experiment.zip)

**Reply by olafmersmann** [reply]
This looks like the standard example experiment? What did you change?
